app/views/client/_pet_list.slim

```diff
@@ -16,7 +16,7 @@
               a href='/pet/#{pet.id}'
                 span.glyphicon.glyphicon-pencil
             td
-              strong = "#{pet.pet_type.pet_type}"
+              strong = "#{pet.pet_type.try(:pet_type)}"
```

Comment:
Seems like the new LookupBy module could handle stuff like this.  Future cleanup